portality/scripts/admin_journalcsv.py

```diff
@@ -12,11 +12,12 @@
 
     parser = argparse.ArgumentParser()
     parser.add_argument("-o", "--out", help="output file for admin csv")
+    parser.add_argument("-a", "--obscure_accounts", help="true if accounts should be obscured", default="true")
```

Comment:
You can make this a flag with no argument to consume, and set `store_true` as the action.  That means you just need to set `-a` on its own, and then `args.obscure_accounts` will be boolean: https://docs.python.org/3/library/argparse.html

Reply:
I set the action to `store_false` to keep the default behaviour (DO obscure the accounts when the flag is NOT set)